docs/book/src/topics/powervs/external-cloud-provider.md

```diff
@@ -0,0 +1,30 @@
+# IBM Power VS External Cloud Provider
+
+## Steps
+
+- To deploy a Power VS workload cluster with IBM Power VS external cloud provider, create a cluster configuration with the [external cloud provider template](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/templates/cluster-template-powervs-cloud-provider.yaml)
+- The [external cloud provider template](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/templates/cluster-template-powervs-cloud-provider.yaml) will use [clusterresourceset](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) and will create the necessary config map, secret and roles to run the cloud controller manager
+- As a prerequisite set the [`powervs-provider-id-fmt`](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/64c9e1d17f1733c721f45a559edba3f4b712bcb0/main.go#L220) flag with value v2
+  
+  ```
+  IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
+  IBMPOWERVS_VIP="192.168.151.22" \
+  IBMPOWERVS_VIP_EXTERNAL="158.175.162.22" \
+  IBMPOWERVS_VIP_CIDR="29" \
+  IBMPOWERVS_IMAGE_NAME="capibm-powervs-centos-8-1-22-4" \
+  IBMPOWERVS_SERVICE_INSTANCE_ID="7845d372-d4e1-46b8-91fc-41051c984601" \
+  IBMPOWERVS_NETWORK_NAME="capi-test-3" \
+  IBMACCOUNT_ID="1234567889" \
+  IBMPOWERVS_RESOURCE_GROUP="powervs-resource-group-name" \
+  IBMVPC_SUBNET_NAMES="subnet-name" \
+  IBMVPC_NAME="vpc-name" \
+  IBMVPC_REGION="vpc-region" \
```

Comment:
These aren't required at the moment, can you check what happens if we just miss these values in the config?

Reply:
why `IBMPOWERVS_RESOURCE_GROUP`? why not `IBMVPC_RESOURCE_GROUP`?

Reply:
Update it to `IBMVPC_RESOURCE_GROUP`